AWSO 2.9.0 Docs update (#4257)
SumoLogic/sumologic-documentation: docs/observability/aws/deploy-use-aws-observability/changelog.md

To install or upgrade to the required version of the CloudFormation template, use the URL of the YAML file corresponding to the right version number as the Amazon S3 URL in the AWS CloudFormation section of the AWS Management Console. See [Selecting a stack template](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-console-create-stack-template.html) for more information.

## v2.9.0, 02-Aug-2024

AWS Observability Solution (S3 Link for CloudFormation template): `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/sumologic_observability.master.template.yaml`

AWS Observability Solution (Terraform-based) repo: [sumologic-solution-templates](https://github.com/SumoLogic/sumologic-solution-templates).

New Feature:
* Amazon RDS app - Added support to analyze and monitor RDS MS SQL CloudWatch logs.

> * Amazon RDS app - New monitors added for RDS MS SQL CloudWatch logs. Solution now supports 70 out of box monitors.

Reviewer: ```suggestion
* Amazon RDS app - New monitors added for RDS MS SQL CloudWatch logs. Solution now supports 70 out-of-box monitors.
```

* By default, the solution now sends CloudFormation deployment telemetry to Sumo Logic. Users have the option to opt out of this telemetry.

Updates:
* Reduced the installation time for the AWS Observability solution using the CloudFormation template.
* Integrated the latest SAM versions with AWSO v2.9.0 (CF + TF) to address CVEs and updated Lambda runtimes from Node.js v18.x to v20.x.
* Upgraded to Sumo Logic Terraform provider v2.31.0 for CVE fixes.
* Updated Terraform test code written in Go language from v1.18 to v1.22, addressing CVEs associated with the gRPC module.

Bug Fixes:

> * Resolved an issue where updating from versions lower than v2.8.0 to v2.8.0 was not functioning correctly. Users are now recommended to update directly to v2.9.0 without needing to go through v2.8.0.

Reviewer: ```suggestion
* Resolved an issue where updating from versions lower than v2.8.0 to v2.8.0 was not functioning correctly. Users are now recommended to update directly to v2.9.0, bypassing v2.8.0.
```

## v2.8.0, 17-May-2024 - Yanked

> ### This version has been Yanked. Please deploy/ update to AWS Observabilty v2.9.0 or latest version.

Reviewer: ```suggestion
### This version has been Yanked. Please deploy/update to AWS Observabilty v2.9.0 or latest version.
```
